src/api/helpers/map-CIS-countries/map-CIS-country/index.ts

```diff
@@ -29,7 +29,7 @@ export const mapCisCountry = (country: CisCountry): MappedCisCountry => {
   mapped.canGetAQuoteByEmail = canGetAQuoteByEmail(mapped);
   mapped.cannotGetAQuote = cannotGetAQuote(mapped);
 
-  mapped.canApplyForInsuranceOnline = applyForInsuranceOnline(country.shortTermCoverAvailabilityDesc, mapped.riskCategory);
+  mapped.canApplyForInsuranceOnline = applyForInsuranceOnline(mapped.shortTermCover, mapped.riskCategory);
```

Comment:
Is `shortTermCoverAvailabilityDesc` no longer relevant here - is `shortTermCover` the mapping result of `shortTermCoverAvailabilityDesc`?

Reply:
yes it is mapped on line 23